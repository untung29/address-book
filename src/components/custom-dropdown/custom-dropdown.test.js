import React from "react";
import renderer from "react-test-renderer";
import CustomDropdown from "./custom-dropdown.component";

it("<CustomDropdown /> Snapshot Testing", () => {
  const CustomDropdownTest = renderer
    .create(<CustomDropdown list={["Descending", "Ascending"]} placeholder="Select to filter by" />)
    .toJSON();
  expect(CustomDropdownTest).toMatchSnapshot();
});
