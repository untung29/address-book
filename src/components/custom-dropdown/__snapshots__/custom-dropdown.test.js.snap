// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> Snapshot Testing 1`] = `
<div
  className="Dropdown-root"
>
  <div
    aria-haspopup="listbox"
    className="Dropdown-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="Dropdown-placeholder"
    >
      Select to filter by
    </div>
    <div
      className="Dropdown-arrow-wrapper"
    >
      <span
        className="Dropdown-arrow"
      />
    </div>
  </div>
</div>
`;
